client/src/components/pages/profileBusiness/index.js

```diff
@@ -0,0 +1,274 @@
+import React from 'react'
+import { Form, Input, AutoComplete, Cascader, Select } from 'antd'
+
+import Button from 'components/utils/Button'
+import { residences, BusinesTypeValues } from './static'
+import Options from "components/utils/option"
+import json from './country.json'
+
+const AutoCompleteOption = AutoComplete.Option;
+
+class BusinessForm extends React.Component {
+
+  state = {
+    autoCompleteResultWebsite: [],
+    autoCompleteResultCuntry: [],
+    autoCompleteResultCity: [],
+    country: ''
+  };
+
+  handleWebsiteChange = value => {
+    let autoCompleteResultWebsite;
+    if (!value) {
+      autoCompleteResultWebsite = [];
+    } else {
+      autoCompleteResultWebsite = ['.com', '.org', '.net'].map(domain => `${value}${domain}`);
+    }
+    this.setState({ autoCompleteResultWebsite });
+  };
+
+  handleCuntryChange = value => {
+    let autoCompleteResultCuntry;
+    if (!value) {
+      autoCompleteResultCuntry = [];
+    } else {
+      const countries = Object.keys(json)
+      let data = countries.filter((country) => {
+        return country.toLowerCase().startsWith(value.toLowerCase())
+      })
+      autoCompleteResultCuntry = data.map(domain => `${domain}`);
```

Comment:
@AbdallahAmmar96 This map isn't doing anything so you could get rid of it (.. then you would have `autoCompleteResultCountry = countries.filter(....`)